include/trace/events/error_report.h

```diff
@@ -0,0 +1,30 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+#undef TRACE_SYSTEM
+#define TRACE_SYSTEM error_report
+
+#if !defined(_TRACE_ERROR_REPORT_H) || defined(TRACE_HEADER_MULTI_READ)
+#define _TRACE_ERROR_REPORT_H
+
+#include <linux/tracepoint.h>
+
+DECLARE_EVENT_CLASS(error_report_template,
+		    TP_PROTO(const char *tool_name, unsigned long id),
```

Comment:
Instead of 'tool_name', would just 'error_detector' be more descriptive? (Without _name, as it's almost implicit from const char* type.)

Reply:
Done. Still gonna mention "tool" in the descriptions though.
(compared to "id", "error_detector" looks a bit bulky in the doc comment, but I don't think that's much of a problem)